main.go

```diff
@@ -143,8 +143,14 @@ func monitorAndRestartProcess() {
 		if time.Since(lastRequestTime) > restartThreshold {
 			logInfo("No requests were received for more than %s. Restarting process...", restartThreshold.String())
 
-			// Kill process
-			killCmd := exec.Command("tskill", strconv.Itoa(processID))
+			// Kill process using tskill or taskkill
+			var killCmd *exec.Cmd
+			if isCommandAvailable("tskill") {
+				killCmd = exec.Command("tskill", strconv.Itoa(processID))
+			} else {
+				killCmd = exec.Command("taskkill", "/F", "/PID", strconv.Itoa(processID))
```

Comment:
```suggestion
				killCmd = exec.Command("taskkill", "/F", "/T", "/PID", strconv.Itoa(processID))
```
This should ensure that all subprocesses get killed, too:
https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/taskkill

Reply:
I was accepting the change on mobile and accidentally committed it without changing the commit message 😐

Reply:
No worries. I merged all changed as one commit and only kept the relevant commit messages in the merged one. I hope that's OK for you - the code is still bound to your user so no credits get lost 🙂